src/debug/debugInfo.ts

```diff
@@ -0,0 +1,32 @@
+/*-----------------------------------------------------------------------------------------------
```

Comment:
In the project you forgot to delete debugInfo.ts and debugInfoProvider outside the debug folder.

Reply:
yes, rebasing error. thx for pointing out.